pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -521,68 +521,6 @@ func TestPodFitsResources(t *testing.T) {
 
 }
 
-func TestPodFitsHost(t *testing.T) {
```

Comment:
sorry for the confusion, I suggested that we copy not move the unit tests, I know it seems redundant now, but that will soon be cleaned up automatically when those predicate functions themselves get deprecated.

Reply:
done